CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 ## Unreleased
+#### Frontend
+ - `v0.92.0` (DL-5790): https://github.com/lblod/frontend-loket/blob/development/CHANGELOG.md#v0920-2024-04-04
```

Comment:
The controle image no longer exists since we converted everything to runtime checks. Should we already remove the controle setup when deploying, or stick with it until we are sure the roles are set up correctly for the admin users?